index.d.ts

```diff
@@ -515,69 +520,69 @@ declare namespace plugins {
    * This plugin automatically instruments the
    * [amqp10](https://github.com/noodlefrenzy/node-amqp10) module.
    */
-  interface amqp10 extends Integration {}
+  interface amqp10 extends Integration { }
```

Comment:
Unnecessary change. If formatting should be reviewed, we should add tooling in the project.